lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/resources.py

```diff
@@ -0,0 +1,60 @@
+# Resource object code (Python 3)
```

Comment:
this file was generated by compiling the resources.qrc file using the command:
`pyside6-rcc .\resources.qrc -o resources.py`